src/IAM/UI/REST/Controller/TokenController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Twitter\IAM\UI\REST\Controller;
+
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\UnsupportedMediaTypeHttpException;
+use Symfony\Component\Routing\Attribute\Route;
+use Twitter\IAM\Application\Logout\LogoutService;
+use Twitter\IAM\Domain\Auth\Exception\AuthUnauthorizedException;
+use Twitter\IAM\Domain\Auth\Exception\ValidationErrorException;
+use Twitter\IAM\Infrastructure\Security\UserIdAwareInterface;
+
+final class TokenController
```

Comment:
All properties are read-only, so maybe make the controller read-only as well?

```suggestion
final readonly class RevokeRefreshTokenController
```